uint32 goal_number
string action
string object_type
geometry_msgs/Pose grasp
dviz_core/Waypoint[] waypoints
